src/Initializer/MemoryManager.cpp

```diff
@@ -660,8 +660,13 @@ void seissol::initializers::MemoryManager::initializeEasiBoundaryReader(const ch
 bool seissol::initializers::isAtElasticAcousticInterface(CellMaterialData &material, unsigned int face) {
   // We define the interface cells as all cells that are in the elastic domain but have a
   // neighbor with acoustic material.
+  //TODO: implement more general coupling
```

Comment:
at name to todo, otherwise no one feels responsible
(feel free to add my name to this)